web-client/public/js/update-app.js

```diff
@@ -447,6 +482,10 @@ const updateBottomDataset = () => {
     updaters.renderNodeColoring();
 };
 
+if (grnState.genePageData.identified === false) {
```

Comment:
We typically don’t directly compare to `true` or `false`—instead, best practice is to try to let the expression itself sound like a `true` or `false` value:
```suggestion
if (!grnState.genePageData.identified) {
```
Note how that sounds like “if not identified” (presumption of course is that the reader knows that `!` means “not”—which they should, if they know the language).